.gitignore

```diff
@@ -1,5 +1,4 @@
 # Build Folders (you can keep bin if you'd like, to store dlls and pdbs)
-[Bb]in/
```

Comment:
I would prefer not to include bin by default, as this will likely be an issue if certain projects are built that aren't hooked up into our default project targets